src/utils.ts

```diff
@@ -173,6 +193,35 @@ export async function resolveLabel(label: string, config: Config): Promise<strin
   return config.provider.resolveName(`${label}.${config.registrar.domain}`);
 }
 
+export async function lookupAddress(address: string, did: string | null, config: Config): Promise<Lookup>  {
+  // First get eventual ENS Registration
+  let label: string | null = null;
+  try {
+    label = await config.provider.lookupAddress(address);
+  } catch (error) {
+    console.log("LookupAddress: Not able to get ENS Name, trying with IDX");
+  }
+  if (label && await config.provider.resolveName(label)) {
+    return {type: "ens", content: await getRegistration(label, config) as Registration };
+  }
```

Comment:
I think we can move this block into the `try {}` block, and the `label` as well.

Reply:
27e9a28f819fb09a9c4ea3796d96362b28f89658 solves this